tests/src/system/basic/WskBasicTests.scala

```diff
@@ -166,6 +168,86 @@ class WskBasicTests
             stdout should not include regex(""""key": "xxx"""")
     }
 
+    it should "create, and get a package to verify annotation parsing" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "packageAnnotations"
+
+            assetHelper.withCleaner(wsk.pkg, name) {
+                (pkg, _) =>
+                    pkg.create(name, annotations = getValidJSONTestArgInput)
+            }
+
+            val stdout = wsk.pkg.get(name).stdout
+            assert(stdout.startsWith(s"ok: got package $name\n"))
+
+            val resJSON = stdout.substring(stdout.indexOf("\n") + 1).parseJson.asJsObject
+            assert(resJSON.fields("annotations") == getValidJSONTestArgOutput)
+    }
+
+    it should "create, and get a package to verify parameter parsing" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "packageParameters"
+
+            assetHelper.withCleaner(wsk.pkg, name) {
+                (pkg, _) =>
+                    pkg.create(name, parameters = getValidJSONTestArgInput)
+            }
+
+            val stdout = wsk.pkg.get(name).stdout
+            assert(stdout.startsWith(s"ok: got package $name\n"))
```

Comment:
you could move this into `WskPackage` and guard with `if (expectedExitCode == SUCCESS) then assert`.